tools/oesign/main.c

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+#include <getopt.h>
```

Comment:
@johnkord I don't recall if glibc remained a dependency that we required in our ansible setup since we default to using clang tools. Can we just double check that this doesn't introduce something that we don't handle in our various deployment configurations (or can safely assume 16.04/18.04 defaults handle), particularly in the manual SDK deployment instructions?

Reply:
@CodeMonkeyLeet we forgot `getopt_long` is a GNU extension and not POSIX compliant.

We added https://github.com/Microsoft/openenclave/pull/1602/files#diff-84d2379234682ee251b1f2f2f0a764e6R6 to check for its existence.

We tested by force compiling and linking on Linux as well.